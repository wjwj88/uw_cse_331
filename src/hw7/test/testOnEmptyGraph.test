# Find a path with Dijkstra's algorithm when there is no graph at all hw7

CreateGraph characters

FindPath characters A B