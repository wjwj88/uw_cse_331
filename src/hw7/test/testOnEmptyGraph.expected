# Find a path with Dijkstra's algorithm when there is no graph at all hw7

created graph characters

unknown character A
unknown character B